exercises/tournament/tests/tournament.rs

```diff
@@ -1,51 +1,57 @@
-use std::fs::File;
-use std::path::Path;
-use std::io::Read;
-
 extern crate tournament;
 
-fn file_equal(output_file: &str, expected_file: &str) {
-    let output = match File::open(&Path::new(output_file)) {
-        Err(e) => panic!("Couldn't open {}: {}", output_file, e),
-        Ok(mut f) => {
-            let mut buf = String::new();
-            match f.read_to_string(&mut buf) {
-                Err(e) => panic!("Couldn't read {}: {}", output_file, e),
-                Ok(_) => buf
-            }
-        }
-    };
-    let expected = match File::open(&Path::new(expected_file)) {
-        Err(e) => panic!("Couldn't open {}: {}", expected_file, e),
-        Ok(mut f) => {
-            let mut buf = String::new();
-            match f.read_to_string(&mut buf) {
-                Err(e) => panic!("Couldn't read {}: {}", expected_file, e),
-                Ok(_) => buf
-            }
-        }
-    };
-    assert_eq!("\n".to_string() + output.as_ref(), "\n".to_string() + expected.as_ref());
-    
-}
-
-
 #[test]
 fn test_good() {
-    assert_eq!(tournament::tally(&Path::new("tests/input1.txt"), &Path::new("tests/output1.txt")).unwrap(), 6);
-    file_equal("tests/output1.txt", "tests/expected1.txt");
+    let input = "Allegoric Alaskians;Blithering Badgers;win\n\
```

Comment:
your comment is accepted on whether you prefer to see `\n\` at the end of every line or not, but if you choose "not", then all other lines will be unindented. Here's an example of what it looks like.

```rust
fn test_good() {
    let input = "Allegoric Alaskians;Blithering Badgers;win
Devastating Donkeys;Courageous Californians;draw";
    let expected = "Team                           | MP |  W |  D |  L |  P
Allegoric Alaskians            |  1 |  1 |  0 |  0 |  3"; // other expected lines omitted, this is just an example
    assert_eq!(tournament::tally(&input), expected);
```